qml/js/helpers/video/YouTubeHelper.js

```diff
@@ -24,7 +24,7 @@ var __wp_youtubehelper__ = {
     },
 
     grabVideoThumbnail: function(videoid) {
-       return "http://img.youtube.com/vi/" + videoid + "/0.jpg";
+       return "https://img.youtube.com/vi/" + videoid + "/0.jpg";
```

Comment:
I have tried in my browser and it loads the video thumbnail as before, so this one is ok!